### ISSUE TITLE
Example code from readme does not work. Forbidden response for session.post.

### ISSUE THREAD

**Issue by Aayushshah196** [parkerhancock/patent_client#98]
I am getting the error mentioned in [#74](https://github.com/parkerhancock/patent_client/issues/74) for version 3.2.2. Can anyone help me out for this case. As I was going through the code, I found that the request for the session_id was giving
 forbidden response.
 Thanks.
![Screenshot from 2023-03-27 20-33-21](https://user-images.githubusercontent.com/50726045/227976131-d8aa511d-4f33-49df-b1c9-52c38c57ec31.png)
![Screenshot from 2023-03-27 20-33-45](https://user-images.githubusercontent.com/50726045/227976269-e485a891-3157-4336-9048-e4ef67eeb1dd.png)

**Comment by mgh5225** [comment]
I dived into the source code and found out that the problem relies on this section of code in `session.py`
`self.headers["User-Agent"] = f"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)"`
which is used in `api.py` :
```
 def get_session(self):
        url = "https://ppubs.uspto.gov/dirsearch-public/users/me/session"
        response = session.post(url, json=str(random.randint(10000, 99999)))
        print(response)
        self.session = response.json()
        self.case_id = self.session["userCase"]["caseId"]
        return self.session
```

**Comment by gambastyle** [comment]
I also have this error, any solution?

**Comment by parkerhancock** [comment]
Closed as duplicate of #101
